tools/resources/README.md

```diff
@@ -0,0 +1,111 @@
+# Data Commons Material UI Icon Generation Tool
+
+## Produce via script
+
+A python script is provided to quickly and easily generate a React component
+that displays a Material UI Icon, an SVG of a Material UI icon for use in the 
+Jinja templates or both.
+
+### Script Usage
+
+To generate both a React component and an SVG of a Material UI icon, run the
+following command:
+
+```bash
+python3 generate_icon.py {icon_name}
```

Comment:
Can you add a "run.sh" wrapper for this script? That will ensure dependencies get installed before running the tool. See some other scripts in the tools project for some examples, but it should look something like:

```
python3 -m venv .env
source .env/bin/activate
python3 -m pip install --upgrade pip setuptools
python3 -m pip install -r requirements.txt
python3 generate_icon.py "$@"
```